R/tostring.R

```diff
@@ -666,6 +667,12 @@ setMethod("toString", "MatrixPrintForm", function(x,
   footer_txt
 }
 
+reorder_ref_fnotes <- function(fns) {
+  ind <- gsub("\\{(.*)\\}.*", "\\1", fns)
+  ind_num <- suppressWarnings(as.numeric(ind))
```

Comment:
I think we can better manage the exception here where it is not numeric. I think we can add a numeric detector before